test/test.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-env node */
+/* eslint-disable no-console */
+var count, config,
+	fs = require( 'fs' ),
+
+	validFixturesFile, validFixtures,
+	assert = require( 'assert' ),
+
+	invalidFixturesFile, invalidFixtures, testPositivesFailures,
+	expectedFailuresFile, expectedFailures,
+	engine, report, results, prevFilename, prevLine,
+	assertDiff = require( 'assert-diff' ),
+	eslint = require( 'eslint' ),
```

Comment:
I don't think we've written it down yet for Node code, but I believe we've been following the Python-esque convention of grouping the import/require statements together as the first group (in our grunt tasks, build scripts, as well as in services). That also matches what most other organisations and open source projects follow (even for otherwise different coding styles).

All other variables can still be grouped logically, but the imports are typically their own group.